examples/asyncio/tornado_asyncio.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+import zmq.asyncio
+from tornado.platform.asyncio import AsyncIOMainLoop
+AsyncIOMainLoop().install()
+
+# This must be instantiated after the installing the IOLoop
```

Comment:
I think you still need to do `zmq.asyncio.install()` to register pyzmq as the asyncio eventloop.